website/pages/docs/connect/proxies/envoy.mdx

```diff
@@ -277,6 +277,17 @@ definition](/docs/connect/registration/service-registration) or
     since HTTP/2 has many requests per connection. For this configuration to be
     respected, a L7 protocol must be defined in the `protocol` field.
 
+- `passive_health_check` - Settings for the load balancer, used to remove
```

Comment:
```suggestion
- `passive_health_check` - Settings for the proxy, used to remove
```

nit, but I would use the word proxy instead of load balancer so as not to introduce another noun into the mix